### ISSUE TITLE
Reports of Brave Talk not functioning with iOS for users with "lockdown mode" enabled 

### ISSUE THREAD

**Issue by lukemulks** [brave/brave-talk#1502]
Received a report from an iOS user that Brave Talk is not functional for users on iOS with "lockdown mode" enabled. 

Screenshots below. 

![Screenshot_20240918_173139_Signal](https://github.com/user-attachments/assets/d689ff76-f3e5-45ae-a362-72a5366eb3a3)
![Screenshot_20240918_173153_Signal](https://github.com/user-attachments/assets/bbbe2b41-cff9-4b28-8e17-5cf58573b0df)

**Comment by mrose17** [comment]
@srirambv - I couldn't reproduce this. if you have some time, could you try to get a STR and maybe make a video? i can see that lockdown mode would impact webassembly, but i didn't see any issues when i tried it.

**Comment by srirambv** [comment]
I can't get Talk to work. I get a message that Lockdown mode is enabled which prevents it to work correctly
<img width=400 src=https://github.com/user-attachments/assets/402f323a-0f24-44f4-95a5-acce5ffb5147>

**Comment by mrose17** [comment]
Thank you! I wonder why I can't reproduce it. Apparently you can join a call and it works, but we have an explicit check when you go create a call. We put that in when the feature first came out. The fact that it appears to work when you join tells me that something changed in the iOS' Lockdown Mode (or we were too conservative initially). I will need to investigate further.

Many thanks for putting a second set of eyes on this.

cc: @mattmcalister

**Comment by mrose17** [comment]
Current status: we need WASM to create a call, but not to join one. We are investigating whether, for creating a call, we can find an alternative. This is gated on another task dealing with the same issue, so this issue is presently blocked.
